conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -12,8 +12,12 @@
 # Add all the application routes to the prod.routes file
 ->         /                          prod.Routes
 
-GET   /direct-debit-verify-email-address/test-only/start     uk.gov.hmrc.directdebitupdateemailfrontend.testOnly.controllers.StartJourneyController.startJourney
-POST  /direct-debit-verify-email-address/test-only/start     uk.gov.hmrc.directdebitupdateemailfrontend.testOnly.controllers.StartJourneyController.startJourneySubmit
+GET   /direct-debit-verify-email/test-only/start     uk.gov.hmrc.directdebitupdateemailfrontend.testOnly.controllers.StartJourneyController.startJourney
```

Comment:
I'm not sure how the test-only page is working in the test environments atm. It was configured here with the service name `/direct-debit-verify-email-address` in this file, but mdtp-frontend-routes has `/direct-debit-verify-email 🤔  It should be without `-address` on the end though